libraries/AP_Motors/scripts/MotorMatrix_setup.lua

```diff
@@ -0,0 +1,9 @@
+-- This script is an example setting up a custom motor matrix mix
```

Comment:
Should this be in the AP_Scripting/examples directory?

Reply:
Yep, moved